src/connector/codec/src/decoder/avro/schema.rs

```diff
@@ -72,11 +72,10 @@ impl MapHandling {
 
 /// This function expects original schema (with `Ref`).
 /// TODO: change `map_handling` to some `Config`, and also unify debezium.
-/// TODO: use `ColumnDesc` in common instead of PB.
 pub fn avro_schema_to_column_descs(
```

Comment:
Shall we also rename the function? Also for json and protobuf.

Reply:
Good catch! 😂